addons/website/static/src/builder/plugins/translation_tab/customize_translation_tab_plugin.js

```diff
@@ -13,17 +13,30 @@ import { TranslateWebpageOption } from "./translate_webpage_option";
  */
 class TranslateToAction extends BuilderAction {
     static id = "translateWebpageAI";
-    static dependencies = ["customizeTranslationTab"];
+    static dependencies = ["customizeTranslationTab", "translation"];
 
     async apply() {
         const translationState = this.dependencies.customizeTranslationTab.getTranslationState();
+        this.elToTranslationInfoMap = this.dependencies.translation.getElToTranslationInfoMap();
         try {
             translationState.isTranslating = true;
             const language = this.services.website.currentWebsite.metadata.langName;
-            const { translationChunks, translationMap } = this.generateTranslationChunks(this.editable);
-            if (translationChunks) {
-                const responses = await this.runTranslationChunks(translationChunks, language);
-                const failedNodeCount = this.applyTranslationsToDOM(translationMap, responses);
+            const { translationChunks, translationMap } = this.generateTranslationChunks(
```

Comment:
```suggestion
            const { translatableChunks, translationMap } = this.generateTranslationChunks(
```
What do you think about this rename?